### DISCUSSION TITLE
Users should be able to verify whether a service has blocked AnonAddy.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#484] [Polls]
Occasionally, someone wants to check why their e-mail address is considered invalid by a service (see [`reddit.com/r/AnonAddy/comments/lpi635`](https://www.reddit.com/r/AnonAddy/comments/lpi635/services_known_to_block_anonaddy_default_domains/?utm_source=share&utm_medium=web2x&context=3)). Would be great if there were a wiki that included a list of blocked services.

**Comment by GHolk** [comment]
Fine, I will manage to create a simple wiki with github wiki soon,
since the founder are not gonna do this.

Pros: github wiki is very easy to setup, ad-free and a lot of people have github account.

Any other suggestion?

**Reply by roke-julian-lockhart** [reply]
> I will manage to create a simple wiki with github wiki soon

@GHolk, thanks. I'll assist in maintenance where I can.

> Any other suggestion?

Not that I can think of. If you enable Discussions and Issues on the repository, I'll posit any suggestions in the Discussions tab.

**Reply by GHolk** [reply]
A basic wiki here:
https://github.com/GHolk/addy-io-unofficial/wiki

**Comment by MrSuddenJoy** [comment]
> since the founder are not gonna do this.

@GHolk why is this?

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/484#discussioncomment-11277062

@MrSuddenJoy, I presume it was an observation erroneously stated as fact, rather than something known.
